src/python/pants/backend/kotlin/lint/ktlint/rules_integration_test.py

```diff
@@ -59,23 +59,23 @@ def rule_runner() -> RuleRunner:
 package org.pantsbuild.example
 
 open class Foo {
-    val CONSTANT = "Constant changes"
+    val constant = "Constant changes"
```

Comment:
there is a new linter rule (`standard:property-naming`) that enforces property names to start with a lowercase letter and use camel case style.
https://pinterest.github.io/ktlint/latest/rules/standard/#property-naming